### ISSUE TITLE
Ignore static fields in SpringObjectBuilder

### ISSUE THREAD

**Issue by stagr** [fabiomaffioletti/jsondoc#207]
When using JSONDoc with Spring MVC it automatically generates documentation also for "static" fields causing the UI to show them in the list of fields and in the prefilled JSON in the playground.

Example:

```
public class Pojo {
  ...
  private static final long serialVersionUID = 1234;
  ...
}
```

generates the following in the list of fields of Pojo class:

`serialVersionUID   Type: long`

and the following in the prefilled JSON in the playground:

```
{
  ...
  "serialVersionUID": 0,
  ...
}
```

A solution is simply to ignore all static fields when iterating over all fields in the SpringObjectBuilder:

```
for (Field field : clazz.getDeclaredFields()) {

  // Only add fields that are not static
  if (!Modifier.isStatic(field.getModifiers())) {
    ApiObjectFieldDoc fieldDoc = new ApiObjectFieldDoc();
   ...
  }
  ...
}
```

**Comment by MonkeyTonk** [comment]
I think PR #217 of issue #215 by @osigge should solve this problem. Maybe issues #207 and #215 should be merged.

**Comment by fabiomaffioletti** [comment]
Solved with #217
